webapp/src/components/sidebar_right/sidebar_right.jsx

```diff
@@ -150,12 +152,12 @@ export default class SidebarRight extends React.PureComponent {
 
         switch (this.state.list) {
         case MyListName:
-            todos = this.props.allIssues.my || [];
+            todos = this.props.myIssues || [];
```

Comment:
These empty arrays are going to cause re-renders. But `this.props.myIssues` should always be defined anyway right? So these actually don't have an effect. Either way I think that this component shouldn't be responsible for this check